cluster/saltbase/salt/kubelet/default

```diff
@@ -151,6 +151,8 @@
   {% set network_plugin = "--network-plugin=opencontrail" %}
 {% elif pillar.get('network_provider', '').lower() == 'cni' %}
   {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/" %}
+{%elif pillar.get('policy_provider', '').lower() == 'calico' and grains['roles'][0] != 'kubernetes-master' -%}
```

Comment:
I think it's probably easiest to jump on a hangout to discuss, as @lxpollitt suggested.  Do you have time for a quick call sometime this week?  @lxpollitt is traveling today, but tomorrow would be good.  Can chat on Slack to organize.